allennlp/predictors/predictor.py

```diff
@@ -239,6 +241,7 @@ def from_path(
         predictor_name: str = None,
         cuda_device: int = -1,
         dataset_reader_to_load: str = "validation",
+        frozen: bool = False,
```

Comment:
You have the default as `False` here, but `True` below.  I think setting this to `True` by default should be fine.